board/views.py

```diff
@@ -35,6 +37,30 @@ class GetPosts(View):
     requests for more post information and the server responds correspondingly.
     """
 
+    def get_post_dict(post):
+        """
+        Return a formated dictionary of a post.
+
+        Args:
```

Comment:
WIll we be using `Args` to specify arguments?

Reply:
i actually like `Params` better, let's use that instead